timon/plugins/http_server/views.py

```diff
@@ -52,6 +67,18 @@ async def get_resources():
     return rsrc_infos
 
 
+@app.route("/heap/")
+async def get_heap():
+    """
+    returns the lenght of the heap
+    """
+    heap = app.tmoncfg.get_queue().heap
+    items_in_heap = []
+    for item in heap:
+        items_in_heap.append(item)
```

Comment:
you might benchmark, not sure what's faster.

I would expected, that getting the heap as a list and sorting will be faster

Reply:
right

Reply:
changed